ops/ccs-ops-misc/tls-certificates/make_bfd_keystores.sh

```diff
@@ -13,22 +13,22 @@ function gen_keystore {
         -validity 730 -keypass changeit -keystore "$3-bluebutton-appserver-keystore.jks" -storepass changeit
 
     echo "Extracting public cert..."
-    keytool -export -keystore "$3-bluebutton-appserver-keystore.jks" -alias server -storepass changeit -file "$3-bluebutton-appserver-public-cert.pem" -rfc 
+    keytool -export -keystore "$3-bluebutton-appserver-keystore.jks" -storetype jks -alias server -storepass changeit -file "$3-bluebutton-appserver-public-cert.pem" -rfc 
 }
 
 # Prod
 gen_keystore "cn=bfd.cms.gov" \
-    "san=dns:prod.bfdcloud.net,dns:mct.prod.bfdcloud.net,dns:dpc.prod.bfdcloud.net,dns:bcda.prod.bfdcloud.net,dns:bb.prod.bfdcloud.net,dns:internal-pdcw10lb01-1951212262.us-east-1.elb.amazonaws.com,dns:bfd.cms.gov,dns:prod.bfd.cms.gov" \
+    "san=dns:prod.bfdcloud.net,dns:mct.prod.bfdcloud.net,dns:dpc.prod.bfdcloud.net,dns:bcda.prod.bfdcloud.net,dns:bb.prod.bfdcloud.net,dns:bfd.cms.gov,dns:prod.bfd.cms.gov,dns:internal-bfd-prod-fhir-1481369778.us-east-1.elb.amazonaws.com" \
     "prod" 
 
 # Prod-SBX
 gen_keystore "cn=prod-sbx.bfd.cms.gov" \
-    "san=dns:prod-sbx.bfdcloud.net,dns:mct.prod-sbx.bfdcloud.net,dns:dpc.prod-sbx.bfdcloud.net,dns:bcda.prod-sbx.bfdcloud.net,dns:bb.prod-sbx.bfdcloud.net,dns:internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com,dns:prod-sbx.bfd.cms.gov,dns:fhir.backend.bluebutton.hhsdevcloud.us" \
+    "san=dns:prod-sbx.bfdcloud.net,dns:mct.prod-sbx.bfdcloud.net,dns:dpc.prod-sbx.bfdcloud.net,dns:bcda.prod-sbx.bfdcloud.net,dns:bb.prod-sbx.bfdcloud.net,dns:fhir.backend.bluebutton.hhsdevcloud.us,dns:prod-sbx.bfd.cms.gov,dns:bfd-prod-sbx-fhir-730339399.us-east-1.elb.amazonaws.com" \
```

Comment:
`fhir.backend.bluebutton.hhsdevcloud.us` can probably be removed as well though would have to double check with Karl first.

Reply:
I spoke with Karl and have removed that DNS name from the list.